api/src/main/java/javax/enterprise/inject/spi/BeanAttributes.java

```diff
@@ -67,6 +67,8 @@
 
     /**
      * Determines if the bean is an {@linkplain javax.enterprise.inject.Alternative alternative}.
+     *
+     * If it is an alternative it may may implement {@link Prioritized} to be selected for the entire application with a priority value.
```

Comment:
I don't think it's useless. An author of a custom bean will read this javadoc. But the wording should be something like:

A custom implementation of `Bean` may implement `Prioritized` in order to be selected for the application. `Prioritized.getPriority()` determines the priority used to resolve ambiguities.